### PR TITLE
feat(TNLT-131):Add Paywall portal inside React Articles

### PR DESCRIPTION
**Pull request by themischatzis**
Add Paywall portal inside article-skeleton to fix Paywal component rendering in the wrong place. It's should be rendering before sponsored articles.

https://nidigitalsolutions.jira.com/browse/TNLT-131

Before
Screenshot 2020-01-29 at 15 18 01
Screenshot 2020-01-29 at 15 18 11

After
Screenshot 2020-01-29 at 15 17 42

### CODE REVIEW

File: packages/article-skeleton/src/article-skeleton.web.js [hunk 1]
```diff
@@ -167,7 +168,11 @@ class ArticleSkeleton extends Component {
                       paidContentClassName={paidContentClassName}
                     />
                   )}
-                  <LazyLoad rootMargin={spacing(40)} threshold={0}>
+                  <PaywallPortal
+                    id="paywall-portal-article-footer"
+                    componentName="subscribe-cta"
+                  />
+                  <LazyLoad rootMargin={spacing(10)} threshold={0.5}>
```
Comment (ValGeorgiev) [inline packages/article-skeleton/src/article-skeleton.web.js]: LazyLoad should not be changed. Probably an issue due rebasing

File: packages/article-skeleton/src/paywall-portall.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import get from "lodash.get";
+
+const PaywallPortal = ({ id, componentName }) => {
```
Comment (ValGeorgiev) [inline packages/article-skeleton/src/paywall-portall.js]: Can you add test for this component

File: packages/article-skeleton/src/article-skeleton.web.js (hunk 1 above)
Comment (themischatzis) [reply packages/article-skeleton/src/article-skeleton.web.js]: Probably yes. I'll make the change.

File: packages/article-skeleton/__tests__/web/paywall-portal.test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import PaywallPortal from "../../src/paywall-portal";
+
+describe("Paywall Portal", () => {
+  describe("Should return an empty div with an id of the id that is passed as prop", () => {
+    const props = {
+      id: "id",
+      componentName: "componentName"
+    };
+
+    it("when window is undefiend", () => {
```
Comment (aronshamash) [inline packages/article-skeleton/__tests__/web/paywall-portal.test.js]: small typo

**Pull request merged**